### PR TITLE
[INLONG-791][Doc] Update the overview document for manager

### LINKED ISSUE #791: [Improve][Doc] Update the overview document for manager
### Description

Update the overview document for manager.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #791 

### Motivation

 Update the overview document for manager.

### Modifications

 Update the overview document for manager.

### CODE REVIEW

File: docs/modules/manager/overview.md [hunk 1]
```diff
@@ -30,8 +25,15 @@ sidebar_position: 1
...
-
-## 5 data model
-![](img/datamodel.jpg)
+## Use Process 
+![](img/interactive.png)
+
+## Data Model
+![](img/data_model.png)
+Currently, the manager mainly consists of the following components:
+- InlongGroup:Data Streams Group, it contains multiple data streams, and one Group represents one data business unit.
```
Comment (healchow) [inline docs/modules/manager/overview.md:34]:
```suggestion
- InlongGroup: Data Streams Group, it contains multiple data streams, and one InlongGroup represents one data business unit.
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md [hunk 2]
```diff
@@ -3,35 +3,39 @@ title: 总览
...
+
+![](img/interactive.png)
+
+## 数据模型
+![](img/data_model.png)
+目前 manager 主要由以下几个组件构成:
+- InlongGroup: 数据流组，包含多个数据流，一个 Group 代表一个数据业务单元。
+- InlongStream: 数据流，一个数据流有具体的数据源、数据格式和数据流向。
+- StreamSource: 数据来源，包括文件采集、MySQL 采集等。
+- StreamSink: 数据目标，数据最后流入的地点，包括 Hive、Clickhouse 等。
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md:39]:
```suggestion
- StreamSink: 数据目标，数据最后流入的节点，包括 Hive、ClickHouse 等。
```

File: docs/modules/manager/overview.md [hunk 3]
```diff
@@ -30,8 +25,15 @@ sidebar_position: 1
...
+- InlongStream: Data Stream, a stream has a specific data source, data format and data sink.
+- StreamSource: Data sources, including File collection, MySQL collection, etc.
+- StreamSink: Data targets, including Hive, Clickhouse, and other locations where data ultimately flows into.
```
Comment (healchow) [inline docs/modules/manager/overview.md:37]:
```suggestion
- StreamSink: Data targets, including Hive, ClickHouse, and other locations where data ultimately flows into.
```

File: docs/modules/manager/overview.md (hunk 1 above)
Comment (fuweng11) [reply docs/modules/manager/overview.md]: Done.

File: docs/modules/manager/overview.md (hunk 3 above)
Comment (fuweng11) [reply docs/modules/manager/overview.md]: Done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md (hunk 2 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md]: Done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md [hunk 4]
```diff
@@ -3,35 +3,39 @@ title: 总览
...
+| manager-client-examples | 客户端使用示例              |
+| manager-client-tools | 客户端命令行工具             |
+
+## 使用流程
+
+![](img/interactive.png)
+
+## 数据模型
+![](img/data_model.png)
+目前 InlongManager 主要由以下几个组件构成:
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md:35]:
```suggestion
目前 InLong Manager 主要有以下数据模型:
```

Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/overview.md]: Done.

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**